mlflow/spacy.py

```diff
@@ -0,0 +1,194 @@
+"""
+The ``mlflow.spacy`` module provides an API for logging and loading SpaCy models.
```

Comment:
nit: `SpaCy` -> `spaCy`?

Reply:
Done!

Reply:
The code still contains `spacy`, `Spacy`, and `SpaCy`.  They should be changed to `spaCy`.

Reply:
Thanks, it seems like these casing things are haunting me :) . Hopefully got them all this time.